manifest.toml

```
@@ -0,0 +1,70 @@
+packaging_format = 2
+
+id = "invidious"
+name = "Invidious"
+description.en = "Alternative front-end to YouTube"
+description.fr = "Front-end alternatif à YouTube"
+
+version = "23.02.14~ynh1"
+
+maintainers = ["eric_G"]
+
+[upstream]
+license = "GPL-3.0-only"
+demo = "https://invidious.site/"
+admindoc = "https://docs.invidious.io/"
+userdoc = "https://github.com/iv-org/documentation"
+code = "https://github.com/iv-org/invidious"
+website = "https://invidio.us/"
+
+[integration]
+yunohost = ">= 11.0.9"
+architectures = ["amd64"]
+multi_instance = false
+ldap = false
+sso = false
+disk = "50M"
+ram.build = "2G"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+    full_domain = true
+
+    [install.admin]
+    type = "user"
+
+    [install.init_main_permission]
+    help.en = "If enabled, Invidious will be accessible by people who do not have an account. This can be changed later via the webadmin."
+    help.fr = "Si cette case est cochée, Invidious sera accessible aux personnes n’ayant pas de compte. Vous pourrez changer ceci plus tard via la webadmin."
+    type = "group"
+    default = "visitors"
+
+    [install.language]
+    ask.en = "Choose the application language"
+    ask.fr = "Choisissez la langue de l'application"
+    type = "string"
+    choices = ["de", "en-US", "es", "fr", "it", "nl", "pt-PT"]
+    default = "fr"
+
+[resources]
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.ports]
+
+    [resources.permissions]
+    main.url = "/"
+
+    api.url = "/api"
+    api.show_tile = false
+    api.allowed = ["visitors"]
+    api.protected = true
+
+    [resources.database]
+    type = "postgresql"
+
+    [resources.apt]
+    packages = "apt-transport-https libssl-dev libxml2-dev libyaml-dev libgmp-dev libreadline-dev postgresql librsvg2-bin imagemagick libsqlite3-dev zlib1g-dev libevent-dev pkg-config libpcre3-dev"
```

```suggestion
    [resources.apt]
    packages = "apt-transport-https libssl-dev libxml2-dev libyaml-dev libgmp-dev libreadline-dev postgresql librsvg2-bin imagemagick libsqlite3-dev zlib1g-dev libevent-dev pkg-config libpcre3-dev"

    [resources.database]
    type = "postgresql"
```

So in fact the issue with failing "Provisioning database" is that postgresql is not installed, the `apt` resource should be placed before the `database` resource (might add some warnings about this)